hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -2774,4 +2774,22 @@ public int getStoreRefCount() {
       .filter(sf -> sf.getReader() != null).filter(HStoreFile::isHFile)
       .mapToInt(HStoreFile::getRefCount).sum();
   }
+
+  /**
+   * @return get maximum ref count of storeFile among all HStore Files
+   *   for the HStore
+   */
+  public int getMaxStoreFileRefCount() {
+    int maxStoreFileRefCount = 0;
+    Collection<HStoreFile> hStoreFiles = this.storeEngine.getStoreFileManager()
+      .getStorefiles();
+    for (HStoreFile storeFile : hStoreFiles) {
```

Comment:
return this.storeEngine.getStoreFileManager().getStorefiles().stream()
      .filter(sf -> sf.getReader() != null).filter(HStoreFile::isHFile)
      .mapToInt(HStoreFile::getRefCount).max()
??

Reply:
Yes same thing but with streams, so better